package.json

```diff
@@ -27,31 +25,36 @@
   },
   "files": [
     "/dist",
+    "/src",
     "/schemas"
   ],
   "main": "dist/enigma.js",
+  "module": "src/services/qix/index.js",
```

Comment:
Should be `src/index.js` until we remove the need to call `enigma.getService()`?